tests/unit/test-util/test.desktop-notifier.js

```diff
@@ -4,26 +4,50 @@ import {assert} from 'chai';
 import sinon from 'sinon';
 
 import {desktopNotifications} from '../../../src/util/desktop-notifier';
+import {createLogger} from '../../../src/util/logger';
 
 describe('util/desktop-notifier', () => {
   describe('desktopNotifications()', () => {
+    const expectedNotification = {
+      title: 'web-ext run: title',
+      message: 'message',
+    };
+
     it('is called and creates a message with correct parameters', () => {
       const fakeNotifier = {
         notify: sinon.spy(() => Promise.resolve()),
       };
-
       desktopNotifications(
```

Comment:
This could be better formatted as:

```js
desktopNotification(expectedNotification, {
  ...
});
```

(also in the other places where there is a similar function call).